utils/install.py

```diff
@@ -387,6 +389,10 @@ def compare(self, version2, separator=". |-", ignorecase=True):
     "manylinux2014" + "x86_64" + WASMEDGE_IMAGE_PLUGIN: VersionString("0.13.0"),
     "manylinux2014" + "aarch64" + WASMEDGE_IMAGE_PLUGIN: VersionString("0.13.0"),
     "ubuntu20.04" + "x86_64" + WASMEDGE_IMAGE_PLUGIN: VersionString("0.13.0"),
+    "darwin" + "x86_64" + WASMEDGE_RUSTLS: VersionString("0.13.4"),
```

Comment:
I don't see a release for this, is there?

Reply:
Here: https://github.com/WasmEdge/WasmEdge/releases/tag/0.13.4

Reply:
Ohh, for 0.13.5 it isn't there

Reply:
https://github.com/WasmEdge/WasmEdge/releases/download/0.13.5/WasmEdge-plugin-wasmedge_rustls-0.13.5-darwin_x86_64.tar.gz

Reply:
Ah... The reason is that the macOS-11 CI runner is broken. So, all artifacts are not built and updated. We are still fixing this issue. If it's working again, then the release assets will be updated soon.